docs/prow/security-mitigations.md

```diff
@@ -0,0 +1,23 @@
+# Security Mitigations
+
+## Regularly Rotate all the Secrets
+
+Next rotation date: **01.08.2019**
+
+All the secret used in the production Prow cluster must be rotated regularly. Rotation interval is six months. Follow the [Prow secret management](./prow-secrets-management.md) to create a new keyring and new Secrets. Then, use [Secrets populator](./../../development/tools/cmd/secretspopulator/README.md) to update all the Secrets in the Prow cluster.
+
+## Do not Print Out Secrets
+
+Developers must not write jobs that emit secrets to the output. This can lead to severe problems.
+
+## Offboarding Checklist
+
+When someone with access to the Prow cluster leaves the project, we must apply the following steps to keep our assets secured.
+
+### Remove the person from the Google project
+
+Remove the person from the `kyma-prow` Google project immediately. Follow [this document](https://cloud.google.com/iam/docs/granting-changing-revoking-access) to revoke access.
+
+### Rotate all the Secrets
+
+All the Secrets that were valid when the person was in the project must be rotated. Follow the [Prow secret management](./prow-secrets-management.md) to create a new keyring and new Secrets. Then, use [Secrets populator](./../../development/tools/cmd/secretspopulator/README.md) to update all the Secrets in the Prow cluster.
```

Comment:
Would that be ok?


```
# Obligatory security measures

Read about the obligatory security measures to take on a regular basis and when a Kyma organization member leaves the project.

## Change Secrets regularly

All Secret used in the Prow production cluster must be changed every six months. Follow [Prow secret management](./prow-secrets-management.md) to create a new key ring and new Secrets. Then, use [Secrets populator](./../../development/tools/cmd/secretspopulator/README.md) to change all Secrets in the Prow cluster.

>**NOTE:** The next Secrets change is planned for August 1, 2019.

## Preventive measures

Do not include any Secrets in jobs as this can lead to severe security issues.

## Offboarding checklist

When a Kyma organization member with access to the Prow cluster leaves the project, take the necessary steps to keep Kyma assets secure.

### Remove Google project access

Remove the person from the `kyma-prow` Google project immediately. Follow [this](https://cloud.google.com/iam/docs/granting-changing-revoking-access) document to revoke necessary access.

### Change Secrets

Change all Secrets that were valid when the person was a project member. Follow [Prow secret management](./prow-secrets-management.md) to create a new key ring and new Secrets. Then, use [Secrets populator](./../../development/tools/cmd/secretspopulator/README.md) to change all Secrets in the Prow cluster.

```